analyticsclient/module.py

```diff
@@ -29,22 +28,6 @@ def answer_distribution(self, data_format=DF.JSON):
 
         return self.client.get(path, data_format=data_format)
 
-    def submission_counts(self, module_ids, data_format=DF.JSON):
```

Comment:
Maybe this deletion should be done in a separate commit? Also, is it possible that there is code that  depends on this method?